webpack.config.js

```diff
@@ -16,25 +16,25 @@
 
 const path = require("path");
 const zlib = require("zlib");
-const glob = require("glob");
 const rtlcss = require("rtlcss");
 const CompressionPlugin = require("compression-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
 const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
 const ImageMinimizerPlugin = require("image-minimizer-webpack-plugin");
-const LiveReloadPlugin = require('webpack-livereload-plugin');
+const LiveReloadPlugin = require("webpack-livereload-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 const ProvidePlugin = require("webpack").ProvidePlugin;
 const RemoveEmptyScriptsPlugin = require("webpack-remove-empty-scripts");
-const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
+const {WebpackManifestPlugin} = require("webpack-manifest-plugin");
+const {WebpackLocalisationPlugin, allLocaleData} = require("./webpack.plugin.localize.js");
 
 /* Shared Plugins */
 
 const sharedCompressionPlugins = [
   new CompressionPlugin({
     filename: "[path][base].gz",
     algorithm: "gzip",
-    compressionOptions: { level: 9, memLevel: 9 },
+    compressionOptions: {level: 9, memLevel: 9},
```

Comment:
thought: does our linter not check and enforce these kinds of changes?

Reply:
(might be?) addressed in 7d1e8cbeb8b1123ddc241a852b5235bc4956039c